src/numbers.rs

```diff
@@ -115,3 +115,10 @@ pub fn float_parse_bytes(s: &[u8]) -> IntFloat {
         IntFloat::Int(int_part)
     }
 }
+
+pub fn fractional_digits(bytes: &[u8]) -> usize {
```

Comment:
I think given this doesn't validate the input is actually digits this should be `pub(crate)` at most.

Reply:
Resolving, as I've removed this :)